src/main/java/org/petrax/models/PetType.java

```diff
@@ -0,0 +1,22 @@
+package org.petrax.models;
+
+public enum PetType {
+
+    CAT("Cat"),
+    DOG("Dog"),
+    TURTLE("Turtle"),
+    BIRD("Bird"),
+    FISH("Fish"),
+    REPTILE("Reptile"),
+    SMALL_MAMMAL("Small Mammal"),
```

Comment:
For small mammal and large mammal maybe we should use Other with a blank space to fill in?